internal/pkg/install/prepare.go

```diff
@@ -44,7 +44,7 @@ func Prepare(data *userdata.UserData) (err error) {
 	if len(data.Install.Root.Data) == 0 {
 		// Should probably have a canonical location to fetch rootfs - github?/s3?
 		// need to figure out how to download latest instead of hardcoding
-		data.Install.Root.Data = append(data.Install.Root.Data, "https://github.com/autonomy/talos/releases/download/"+version.Tag+"/rootfs.tar.gz")
+		data.Install.Root.Data = append(data.Install.Root.Data, "https://github.com/autonomy/talos/releases/download/v0.1.0-alpha.19/rootfs.tar.gz")
```

Comment:
Is this left over from testing or do we need to find a mo betta way to handle this for testing?

Reply:
Yeah we need a better way here, mainly because we don't publish any artifacts.